docs/dev/replay-e2e-test.md

```diff
@@ -0,0 +1,49 @@
+# Replay End-to-End Tests
+
+* This document introduces replay tests for those located in [sdk_cli_azure_test](../../src/promptflow/tests/sdk_cli_azure_test/e2etests/) and [sdk_cli_test](../../src/promptflow/tests/sdk_cli_test/e2etests/).
+* The primary purpose of replay tests is to avoid the need for credentials, Azure workspaces, OpenAI tokens, and to directly test PromptFlow behavior.
+* Although there are different techniques behind recording/replaying, there are some common steps to run the tests in replay mode.
+* The key handle of replay tests is the environment variable `PROMPT_FLOW_TEST_MODE`.
+
+## (1 Step Version) How to Run Tests in Replay Mode
```

Comment:
Remove (1 Step Version)

Reply:
How to Run Tests in Replay Mode -> Run tests in replay mode

Only cap the first word